Dockerfile

```
@@ -0,0 +1,38 @@
+FROM ubuntu:18.04
+
+RUN mkdir /data
+
+ENV BBLFSH_HOSTNAME=tmexp_bblfshd \
+  BBLFSH_PORT=9432 \
+  GITBASE_HOSTNAME=gitbase \
```

We should probably prefix it by default like bblfshd to avoid conflict if we run another gitbase:
```suggestion
  GITBASE_HOSTNAME=tmexp_gitbase \
```
This requires some harmonization in docs and maybe some other places though. I'm fine if we do it at some other point, it's not that important.

---

no it's fine, I think it is only here, the Makefile and the docs